### DISCUSSION TITLE
adbc driver use examples 0.8.0 Release

### DISCUSSION THREAD

**Discussion by chrisfw** [duckdb/duckdb#7576] [Q&A]
Hello,

I was excited to read that adbc support has been incorporated into the 0.8.0 duckdb release.  I searched the web and the github project to see if there were any examples, but could not find any for duckdb.  I imagine it is very similar to the sqlite example I found [here](https://arrow.apache.org/blog/2023/01/05/introducing-arrow-adbc/), but it would be great to have a duckdb specific quick start to work from.  Please let me know if there is a resource available to which you can point me.

```
import adbc_driver_sqlite.dbapi

uri = "file:mydb.sqlite"
with adbc_driver_sqlite.dbapi.connect(uri) as conn:
    with conn.cursor() as cur:
        cur.execute("SELECT * FROM customer")
        table = cur.fetch_arrow_table()
        # Process the results
```  

Thank you,
Chris Whelan

**Comment by pdet** [comment]
Maybe the ADBC tests can provide enough guidance? 
https://github.com/duckdb/duckdb/blob/master/test/api/adbc/test_adbc.cpp

**Comment by chrisfw** [comment]
Hi @pdet , Thank you for your quick response.  I was planning to use Python, so though I might be able to figure out the analogous statements from the C++ example, it would be really helpful to have a pythonic example to work from.

Regards,
Chris

**Reply by pdet** [reply]
@Tishj will have a look on pushing the python driver on the following days.

**Comment by chrisfw** [comment]
Thank you @pdet.  I appreciate your help!

Regards,
Chris
